code/modules/arcade/mob_hunt/game_server.dm

```diff
@@ -0,0 +1,164 @@
+var/global/datum/controller/process/mob_hunt/mob_hunt_server
```

Comment:
seed controller isn't really a standard controller; garbage collector is also an exception and not the rule.